tok/xfst-tok-gael.script

```diff
@@ -44,12 +45,12 @@ define WS [SP|NL|TAB];
 
 echo >>> define punctuation
 
-
-#define SINGLE [%" | %. | %, | %; | %: | %! | %? | %( | %) | %[ | %] | %{ | %} |%'| %_ | %` | %# | %& | %£ | %€ | %< | %> |%/ | %° | %· |® | ¡|¿];
+#define SINGLE [%" | %. | %, | %; | %: | %! | %? | %( | %) | %[ | %] | %{ | %} |%'| %_ | %` | %# | %& | %£ | %€ | %< | %> |%/ | %° | %· | %® | %¡ | %¿ ];
 #without underscore
-define SINGLE [%" | %. | %, | %; | %: | %! | %? | %( | %) | %[ | %] | %{ | %} |%'| %` | %# | %& | %£ | %€ | %< | %> |%/ | %° | %· |® | ¡|¿];
+define SINGLE [%" | %. | %, | %; | %: | %! | %? | %( | %) | %[ | %] | %{ | %} | %' | %` | %# | %& | %£ | %€ | %< | %> | %° | %˚ | %º | %· | %® | %¡ | %¿ | %⇒ | %√ | %± | %∝ | %∝ | %α | %Ω | %Ω | %Ω | %ρ | %Ρ | %• | %µ | %π | %∏ | %Π | %π | %ㅠ | %Δ | %∆ | %© | %… | %♠️ | %♥️ | %♦️ | %♣️ | %♕ | %♛ | %♔ | %♖ | %♜ | %ℕ | %† | %① | %② | %③ | %④ | %⑤ | %⑥ | %⑦ | %⑧ | %⑨ | %⑩ | %⑪ | %⑫ | %⑬ | %⑭ | %⑮ | %⑯ | %⑰ | %⑱ | %⑲ | %⑳ | %→ | %↑ | %↓ | %Γ | %ϑ | %Λ | %Ξ | %θ | %Σ | %Φ | %Ψ | %Ω | %ø | %∞ | %∂ | %∧ | %∀ | %∑ | %β | %γ | %δ | %Ε | %ε | %Β | %Α | %Ζ | %ζ | %Η | %η | %Θ | % | %Ι | %ι | %Κ | %κ | %λ | %Μ | %Ν | %ν | %ξ | %Ο | %ο | %σ | %ς | %Τ | %τ | %Υ | %υ | %Φ | %φ | %Χ | %χ | %Ψ | %ψ | %Ω | %ω | %μ | %∴ | %½ | %¼ | %¾ | %⅔ | %⅓ | %¯ | %³ | %² | %¹ | %≈ | %↔  | %➞ | %⇔ | %▶️ | %◀️  | %▼ | %✔ | %✘ | %➡️ | %⬅️ | %⬆️ | %⬇️ | %↗️ | %↘️ | %↙️ | %↖️ | %↕️ | %↔️ | %↪️ | %↩️ | %⤴️ | %⤵️  | %~ | %∕ | %‰  | %¬ |  %∪ | %∩ | %≥ | %≤ | %≡ | %≈ | %≠ | %× | %| | %∠ | %» | %« | %ı | %ƒ | %⁄ | %Ө | % ];
```

Comment:
I think this needs to be done differently, as SINGLE is negated elsewhere, and this change makes it massive